### DISCUSSION TITLE
Ideas and small problem

### DISCUSSION THREAD

**Discussion by benji88i** [bitstower/markets#60] [Ideas]
![Screenshot from 2021-02-04 02-16-16](https://user-images.githubusercontent.com/68906758/106767444-fa8f0c00-668e-11eb-9ed9-bde276b3af24.png)

Hi there,

I love this app and I noticed the source is mostly from yahoo finance. I was curious to know if perhaps the developer would like to integrate data from other sources. I would happy to volunteer with that. 

Quote a lot of the candadian listed stocks are not there (I trade the TSX a lot more)

Also I have an issue with the dark mode. It doesnt seem to work. I am running Fedora 33 with Cinnamon DE

**Comment by tomasz-oponowicz** [comment]
hi @benji88i, nice to meet you

Sorry for the late reply.

>  I was curious to know if perhaps the developer would like to integrate data from other sources

I prefer to keep a single data source because it simplifies things a lot. But I'm curious. 

Do you have a specific data source in mind? Does your data source require API key?

> Also I have an issue with the dark mode. It doesnt seem to work. I am running Fedora 33 with Cinnamon DE

Do you use Flatpak version? Does your theme support dark mode? Please file a bug report.
